### DISCUSSION TITLE
希望增加过滤标签为仅限港澳台的过滤器

### DISCUSSION THREAD

**Discussion by joker1024567** [the1812/Bilibili-Evolved#2492] [功能建议 / Ideas]
对于仅限港澳台的番剧，根据相关的版权法律，在大陆地区的肯定是无效网页，但对于港澳台地区肯定是仅有的播放页
所以，希望有一个一键过滤按钮，可以看到真实的可播放的番剧列表

**Comment by the1812** [comment]
关键词里加一下港澳台不行吗

**Reply by joker1024567** [reply]
就是把港澳台的视频去掉，这要怎么操作

**Reply by the1812** [reply]
动态过滤器里加个港澳台
![image](https://user-images.githubusercontent.com/26504152/140638939-d367cfb6-1226-464c-80c7-952f47bbe097.png)

**Reply by joker1024567** [reply]
这东西在哪儿呢？第一次用这个功能没找到

**Reply by the1812** [reply]
你脚本用的哪个版本

**Reply by joker1024567** [reply]
![image](https://user-images.githubusercontent.com/91998801/140639582-9c0e1049-f8c4-44e0-ac4d-07586aaed1a2.png)

**Reply by the1812** [reply]
设置里把`动态过滤器`打开, 然后进动态首页就有

**Reply by joker1024567** [reply]
这个是只过滤动态消息吗？我是希望有在番剧分区的番剧索引里过滤一下

**Reply by the1812** [reply]
是啊，番剧分区那边过滤不了

**Reply by joker1024567** [reply]
能不能开发一下这个功能，因为港澳台无效的番剧占到了总番剧的三分之一

**Reply by the1812** [reply]
把要看的追下番，以后都在动态里看就行了

**Comment by joker1024567** [comment]
主要是我看到有同一个番剧，一个是有效地，一个是无效的，所以想过滤掉无效的
